outplacement_final_report_send/models/outplacement.py

```diff
@@ -1,11 +1,9 @@
 import datetime
```

Comment:
this module's manifest file hasn't been updated, did you update the manifest file in the wrong module?